packages/dds/tree/src/feature-libraries/sequence-field/format.ts

```diff
@@ -0,0 +1,220 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { JsonableTree } from "../../tree";
+import { NodeChangeset } from "../modular-schema";
+
+export type NodeChangeType = NodeChangeset;
+export type Changeset<TNodeChange = NodeChangeType> = MarkList<TNodeChange>;
+
+export type MarkList<TNodeChange = NodeChangeType, TMark = Mark<TNodeChange>> = TMark[];
+
+export type Mark<TNodeChange = NodeChangeType> =
+	| SizedMark<TNodeChange>
+	| Attach<TNodeChange>;
+
+export type ObjectMark<TNodeChange = NodeChangeType> =
+	| SizedObjectMark<TNodeChange>
+	| Attach<TNodeChange>;
+
+export type SizedMark<TNodeChange = NodeChangeType> =
+	| Skip
+	| SizedObjectMark<TNodeChange>;
+
+export type SizedObjectMark<TNodeChange = NodeChangeType> =
+	| Tomb
+	| Modify<TNodeChange>
+	| Detach
+	| Reattach
+	| ModifyReattach<TNodeChange>
+	| ModifyDetach<TNodeChange>;
+
+export interface Tomb {
+	type: "Tomb";
+	change: ChangesetTag;
+	count: number;
+}
+
+export interface SetValue extends HasOpId {
+	/**
+	 * Can be left unset to represent the value being cleared.
+	 */
```

Comment:
Are your referring to the fact that `JSON.parse(JSON.stringify({ foo: undefined, bar: 42 }))` produces `{ bar: 42 }`?
That would be an issue if `Value` included `undefined` but it does not.
Anyway, there's another issue with this: it's dead code since the value changes are now handled by the `ModularEditBuilder`. :)